app/models/events/steps/personal_details.rb

```diff
@@ -3,12 +3,33 @@ module Steps
     class PersonalDetails < ::GITWizard::Steps::Identity
       attribute :is_walk_in, :boolean
       attribute :event_id
+      attribute :channel_id, :integer
 
       validates :event_id, presence: true
 
+      def channel_ids
```

Comment:
I believe specs will need updating for new methods added ie `channel_ids` and `save`.